tests/suites/test_suite_psa_crypto.function

```diff
@@ -3272,11 +3272,16 @@ void aead_encrypt_decrypt( int key_type_arg, data_t *key_data,
     size_t output_length = 0;
     unsigned char *output_data2 = NULL;
     size_t output_length2 = 0;
-    size_t tag_length = 16;
+    size_t tag_length = PSA_AEAD_TAG_LENGTH( alg );
     psa_status_t expected_result = expected_result_arg;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
 
     output_size = input_data->len + tag_length;
+    /* For all currently defined algorithms, PSA_AEAD_ENCRYPT_OUTPUT_SIZE
+     * should be exact. */
+    if( expected_result != PSA_ERROR_INVALID_ARGUMENT )
```

Comment:
Do we expect the length to be correct for errors other than PSA_ERROR_INVALID_ARGUMENT? If not, should we check for expected_result == PSA_SUCCESS?

Reply:
The length is expected to be correct except if the algorithm is invalid. This includes cases where the operation might fail for other reasons, at least `INVALID_SIGNATURE`. It would be better to have properly separate tests for the macro and for the operation, but that's a much bigger task than this PR (it applies to all the other size macros).